index.html

```diff
@@ -1,48 +1,285 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
-   
-    <header id="nav-bar">
-       <a>sonu</a>
-         <nav>
-
-            <desktop id="header-desktop">
-                <h2 id="portfolio">Portoflio</h2>
-               
-                <h4 id="contact-header">Contact</h4>
-                <img src="images/letter-icon.svg" id="letter-icon">
-               <a href="#"><img src="images/Union.png" id="menu"></a>
-            </desktop>
-         </nav>
-           
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>portfolio</title>
+  </head>
+  <body>
+    <header class="nav">
+      <a href="#">sonu</a>
+      <i class="fa-solid fa-bars"></i>
     </header>
-    <section id="home">
-        
-        <div id="intro">
-        <h1>
-            Hey There. I'm Sonu
-        </h1>
-        <h2>
-            I’am a Software Developer
-        </h2>
+    <section class="main">
+      <h1>Hey There. I`m sonu</h1>
+      <h4>i am a software developer</h4>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <ul>
+        <li>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-solid fa-m"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-solid fa-m"></i></a>
+        </li>
+      </ul>
+    </section>
+
+
+    <section class="work-section">
+      <h1>My Recent Works</h1>

```

Comment:
[Optional]
I'm only marking this as optional since the previous reviewer missed this detail, however, it should be fixed to have a semantically correct project ✨ 
- [ ] There should only be one `h1` in each page which is the main header of the page, so for better semantics kindly change this `h1` and any other following `h1` tag to another `hx` tag 👍